tests/model_registry/rbac/conftest.py

```diff
@@ -89,6 +93,38 @@ def sa_token(service_account: ServiceAccount) -> str:
         raise
 
 
+@pytest.fixture(scope="session", autouse=True)
+def check_env_vars() -> None:
+    """
+    Check if the required environment variables are set.
+    Fails the test session if any are missing.
+    """
+    missing: list[str] = []
+    missing.extend(var for var in ["ADMIN_PASSWORD", "NON_ADMIN_PASSWORD"] if not os.environ.get(var))
+    if missing:
+        pytest.skip(f"Required environment variables not set: {', '.join(missing)}")
```

Comment:
Why do we need to set it? It should be randomly generated in the test session.

Reply:
code written before the idp and user utils